Add Python Turtle workshop (#1340)
hackclub/hackclub: workshops/python turtal/README.md

```python
from turtle import *

forward(100)
right(90)
forward(100)
home()
```

For more information click [hear](https://docs.python.org/3/library/turtle.html#turtle.forward)

## Some basic shapes with turtle

__Triangle__<br />
```python
import turtle
pencil = turtle.Turtle()
pencil.forward(100)
pencil.left(120)
pencil.forward(100)
pencil.left(120)
pencil.forward(100)
turtle.done()
```
![](https://cloud-36kk4uh0h.vercel.app/triangle.gif)

__Star__<br />
```python
import turtle  
  
star = turtle.Turtle() 
  
for i in range(50): 
    star.forward(50) 
    star.right(144) 
      
turtle.done() 
```
![](https://cloud-36kk4uh0h.vercel.app/star.gif)

__Hexagon__<br />
```python
import turtle  
polygon = turtle.Turtle() 
  
num_sides = 6
side_length = 70
angle = 360.0 / num_sides  
  
for i in range(num_sides): 
    polygon.forward(side_length) 
    polygon.right(angle) 
      
turtle.done()  
```
![](https://cloud-36kk4uh0h.vercel.app/hexagon.gif)

## Rainbow Benzene 

> So now you have gain basic knowledge of python turtle library. Now lets build the exicting raindow benzene with turtle.

Reviewer: Grammar/punctuation is off here

---
name: 'Python Turtle'
description: 'Exciting Visuals with Python Turtle'
author: '@YashKalbande'
---

# Exciting Visuals with Python Turtle

## Introduction

> Turtle graphics is a Python feature like a drawing board, which lets us command a turtle to draw all over it. It is a popular way for introducing programming to kids. Virtual turtles can be programmed to move around the screen. The turtle draws lines as it moves. The "turtle" could look like the turtle animal, an arrow, or be invisibile. The user can write turtle programs that draw beautiful shapes and learn to program at the same time.The original Turtle software was developed by Wally Feurzig and Seymour Papert in 1966.<br/>

Reviewer 2: ```suggestion
Turtle graphics is a Python feature like a drawing board, which lets us command a turtle to draw all over it. It is a popular way of introducing programming to kids. Virtual turtles can be programmed to move around the screen. The turtle draws lines as it moves. The "turtle" could look like the turtle animal, an arrow, or be invisible. The user can write turtle programs that draw beautiful shapes and learn to program at the same time. The original Turtle software was developed by Wally Feurzig and Seymour Papert in 1966.<br/>
```

## Quickstart

TODO - Explain how to install and setup Python 3 on [repl.it](https://repl.it/languages/python3) for turtle graphics.

Then enter the following code in newly created main.py file.:

```python
from turtle import *

forward(100)
left(90)
forward(100)
left(90)
forward(100)
left(90)
forward(100)
```

Click on Run Button. The output of this program will look like this:

![](https://cloud-36kk4uh0h.vercel.app/square.gif)

The instructions in your program tell the "turtle" how to move. The turtle draws a line behind it as it moves. This program draws a square. The steps given to the program are:

1. Move forward 100 steps. (In the beginning, the turtle is facing to the right.)
2. Turn 90 degrees to the left.
3. Move forward 100 steps.
4. Turn 90 degrees to the left.
5. Move forward 100 steps.
6. Turn 90 degrees to the left.
7. Move forward 100 steps. The turtle has ended up where it started.

With these seven steps, the turtle draws a square. The `from turtle import *` is an instruction needed at the beginning of all of your turtle programs. It imports the turtle module so you can do the turtle instructions.

> There are many instructions like left() and forward(). These instructions are called functions. This tutorial explains many of the functions in the turtle module. When you learn more of these functions, you will be able to draw many different shapes and beautiful pictures!

Reviewer 2: ```suggestion
There are many instructions like left() and forward(). These instructions are called functions. This tutorial explains many of the functions in the turtle module. When you learn more about these functions, you will be able to draw many different shapes and beautiful pictures!
```

## Turtle Motion

By calling these functions, the turtle can be made to move around the screen. Imagine the turtle holding a pen down on the ground and drawing a line as it moves around.

The turtle's position is two numbers: the X coordinate and Y coordinate.

### forward(*distance*)

> The forward() function moves the turtle *distance* number of steps in the current direction. If the pen is down (see pendown() and penup()) a line will be drawn as the turtle moves forward. If *distance* is a negative number, the turtle will move backwards.

Reviewer 2: ```suggestion
The forward() function moves the turtle *distance* a number of steps in the current direction. If the pen is down (see pendown() and penup()) a line will be drawn as the turtle moves forward. If *distance* is a negative number, the turtle will move backward.
```
